docs/man-pages/condor_token_request_approve.rst

```diff
@@ -0,0 +1,110 @@
+
+
+*condor_token_request_approve*
+======================
+
+approve a token request at a remote daemon
+:index:`condor_token_request_approve<single: condor_token_request_approve; HTCondor commands>`\ :index:`condor_token_request_approve command`
+
+Synopsis
+--------
+
+**condor_token_request_approve** [**-reqid** *val*]
+[**-pool** *pool_name*] [**-name** hostname] [**-type** *type*]
+[**-debug**]
+
+**condor_token_request** [**-help** ]
+
+Description
+-----------
+
+*condor_token_request_approve* will approve an request for an authentication token
+queued at a remote daemon.  Once approved, the requester will be able to fetch a
+fully signed token from the daemon and use it to authenticate with the TOKEN method.
+
+**NOTE** that any user can request a very powerful token, even allowing them to be
+the HTCondor administrator.  Review token requests carefully to ensure you understand
+what authorization you are approving.  The only safe way to approve a request is to
+have the request ID communicated out-of-band, ensuring the request's authenticity.
+
+By default, users can only approve requests for their own identity (that is, a user
+authenticating as ``bucky@cs.wisc.edu`` can only approve token requests for the identity
+``bucky@cs.wisc.edu``).  Users with ``ADMINISTRATOR`` authorization can approve any
+request.
+
+If you want to approve multiple requests at once, do not provide the **-reqid** flag;
+in that case, the utility will iterate through all known requests.
+
+By default, *condor_token_request_approve* will query the local *condor_collector*;
+by specifying a combination of **-pool**, **-name**, or **-type**, the tool can
+request tokens in other pools, on other hosts, or different daemon types.
+
+Options
+-------
+
+ **-debug**
+    Causes debugging information to be sent to ``stderr``, based on the
+    value of the configuration variable ``TOOL_DEBUG``.
+ **-help**
+    Display brief usage information and exit.
+ **-name** *hostname*
+    Request a token from the daemon named *hostname* in the pool.  If not specified,
+    the locally-running daemons will be used.
+ **-pool** *pool_name*
+    Request a token from a daemon in a non-default pool *pool_name*.
+ **-token** *filename*
+    Specifies a filename, relative to the directory in the *SEC_TOKEN_DIRECTORY*
+    configuration variable (defaulting to ``~/.condor/tokens.d``), where
+    the resulting token is stored.  If not specified, the token will be
+    sent to ``stdout``.
+ **-type** *type*
+    Request a token from a specific daemon type *type*.  If not given, a
+    *condor_collector* is used.
+
+Examples
+--------
+
+To approve the tokens at the default *condor_collector*, one-by-one:
+
+::
+
+    % condor_token_request_approve                                                                                               
+    RequestedIdentity = "bucky@cs.wisc.edu"
```

Comment:
Maybe not the best example. In general the admin wouldn't want to approve requests where the RequestedIdentity is a different person than the AuthenticatedIdentity.